ShoppingList/src/main/java/org/openintents/shopping/ui/ShoppingActivity.java

```diff
@@ -1078,7 +1078,7 @@ public boolean onLongClick(View v) {
             @Override
             public void drag(int from, int to) {
                 if (debug) {
-                    Log.v("DRAG", "" + from + "/" + to);
+                    Log.v("DRAG", Integer.toString(from) + "/" + to);
```

Comment:
This makes the code less readable. I prefer not to have this conversion here. As it is in debug and log only, it is preferrable to break the rule.